index.html

```diff
@@ -5,63 +5,75 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>My Portfolio Microverse</title>
+    <title>Yusuf Obr | Portfolio</title>
   </head>
   <body>
-    <!-- Header Section -->
+    <!-- Header -->
     <header class="header">
-      <a href="#">Yusuf Obr</a>
-      <nav class="hamburger">
-        <span class="hamburger-top"></span>
-        <span class="hamburger-middle"></span>
-        <span class="hamburger-bottom"></span>
+      <nav class="nav-desktop">
+        <a href="#">Yusuf Obr</a>
+        <ul class="nav-sections">
+          <li>Portoflio</li>
+          <li>About</li>
+          <li>Contact</li>
```

Comment:
👏 👏 I love your design! 🤟 
- [ ] As a conventional practice and for accessibility purposes, navigation lists are usually wrapped in `a` tags so that users can easily navigate the page for required resources 😉 
_Kindly use nest  the link text in a `a` tag - so that it looks like this:_
```
 <li><a href="#id">Portoflio</a></li>
 <li><a href="#id">About</a></li>
 <li><a href="#id">Contact</a></li>
```

- [ ] [OPTIONAL] I have included an `id` attribute in the `href` attribute of your `a` tag. If you insert the `id`s of all those sections, it will allow users to conveniently move around on your page. 🚧 
- [ ] [OPTIONAL] To take this nice portfolio page to another level, you might consider adding a style for its `:hover` state so that it changes color when users hover on it. You can use the same color style when your button gets hovered on. This will improve your page's accessibility. 🤞 

![image](https://user-images.githubusercontent.com/99715640/220901723-eed452a9-7071-4cfd-b2b6-28cabe1b91be.png)